cli/parse-cli-args/src/index.ts

```diff
@@ -189,6 +189,12 @@ export async function parseCliArgs (
   }
 
   const knownOptions = new Set(Object.keys(types))
+  const allowConvertOptions = ['allow-build']
```

Comment:
if we make it configurable then probably it should be part of the "types", or passed in to the function. But now we group the types by command name, so I need to think about it